gen-ai/orchestrator-server/src/main/python/tock-llm-indexing-tools/generate_dataset.py

```diff
@@ -164,51 +158,33 @@ def init_langfuse():
     return Langfuse()
 
 
-def _send_to_langfuse(dataset: pd.DataFrame, dataset_name: str):
-    # Transforms dataset to JSON format
-    records = dataset.to_json(orient='records')
-    records = loads(str(records))
-
+def _send_to_langfuse(dataset_items: list[DatasetItem], dataset_name: str):
     # Initializes the Langfuse client
     client = init_langfuse()
-
     logging.info('Creating dataset %s on Langfuse...', dataset_name)
-
-    # Creates dataset in Langfuse
     lf_dataset = client.create_dataset(name=dataset_name)
-
     logging.info('Creating examples on Langfuse dataset id %s...', lf_dataset.id)
-
-    # Prepares inputs and outputs
-    inputs = [
-        {
-            'question': r['question'],
-            'locale': r['locale'],
-            'no_answer': r['no_answer'],
+    for item in dataset_items:
+        item_input = {
+            'question': item.question,
+            'locale': item.locale,
+            'no_answer': item.no_answer,
         }
-        for r in records
-    ]
-    metadatas = [{'topic': r['topic']} for r in records]
-
-    outputs = [
-        {
-            'answer': r['answer'],
-            'quality': r['quality'],
+        item_metadata = {
+            'topic': item.topic
+        }
+        item_output = {
+            'answer': item.answer
         }
-        for r in records
-    ]
 
-    # Creates examples in the dataset on Langfuse
-    for input, metadata, output in zip(inputs, metadatas, outputs):
-        logging.info('import data')
+        # Creates examples in the dataset on Langfuse
         client.create_dataset_item(
             dataset_name=dataset_name,
-            input=input,
-            expected_output=output,
-            metadata=metadata,
+            input=item_input,
+            expected_output=item_output,
+            metadata=item_metadata,
         )
 
-
 if __name__ == '__main__':
     cli_args = docopt(__doc__, version='Dataset generator 0.1.0')
     load_dotenv()
```

Comment:
I run it locally an noticed that the .env file is committed, this will lead developpers to commit their configurations can you rename it .example.env for instance and add .env to a local .gitignore file

Reply:
The example file for ‘.env’ is with the other examples in ./examples